system_test/enterprise_e2e/tests/test_switch_fuzz.py

```
@@ -0,0 +1,1442 @@
+        # availability-plane witness (the fixture only starts the process).
+        qwep_connect = (
+            f"ws::addr=127.0.0.1:{p1_ports.http}"
+            ";username=admin;password=quest"
+            ";failover_max_duration_ms=10000"
+            ";auth_timeout_ms=5000;"
+        )
+        try:
+            go_egress_sidecar.connect(qwep_connect)
+            qwep_for_probes = go_egress_sidecar
+            LOG.info("egress QWeP sidecar connected (availability plane)")
+        except Exception as exc:
```

Ruff (0.15.20)

[warning] Do not catch blind exception: `Exception`

(BLE001)